test-utils/runtime/src/system.rs

```diff
@@ -365,7 +365,7 @@ mod tests {
 			Sr25519Keyring::Charlie.to_raw_public()
 		];
 		TestExternalities::new_with_code(
-			WASM_BINARY,
+			WASM_BINARY.unwrap(),
```

Comment:
Maybe we should replace all these `unwrap` calls with a function that is provided by the `test-runtime`. The function should use an `expect` with a good explanation why the wasm binary probably wasn't build. Otherwise I see a lot of issues being opened that will ask about this.